CHANGES.rst

```diff
@@ -79,6 +79,8 @@ astropy.nddata
   when specifying the bit flags to be used or ignored when converting a bit
   field to a boolean. [#10095]
 
+- Added ``reshape_as_blocks`` function. [#10214]
```

Comment:
Can you give more context here as to what it does?

Reply:
I added some more context.  It does exactly what it says on the tin.  :smile: